composer.json

```
@@ -63,6 +63,7 @@
         "twig/twig": "^2.12.1"
     },
     "conflict": {
+        "aws/aws-sdk-php": "<3.0",
```

We don't know what changes the 4.x release can introduce. When it is released, we can remove the conflict if there are no breaking changes.
```suggestion
        "aws/aws-sdk-php": "<3.0 || >= 4.0",
```